runtime/help/commands.md

```diff
@@ -31,9 +31,10 @@ quotes here but these are not necessary when entering the command in micro.
 
 * `quit`: quits micro.
 
-* `goto 'line'`: jumps to the given line number. A negative number can be
-   passed to jump inward from the end of the file; for example, -5 jumps
-   to the 5th-last line in the file.
+* `goto 'line[:col]'`: Jumps to the given line (and column) number.
+   `line` can be prefixed with `+`, `-` or `~`.
+   +/- will be used to perform relative jumps from the current position
+   of the cursor, while ~ will perform a jump inward from the end of the file.
```

Comment:
Nit: may be useful to restore (and extend) the example. Maybe e.g.: `+/- will perform relative jumps from the current position of the cursor (e.g. +5 will jump 5 lines down), while ~ will perform a jump inward from the end of the file (e.g. ~5 jumps to the 5th-last line in the file).`